test/integration/testdata/basic_plugin.yaml

```diff
@@ -0,0 +1,13 @@
+sonobuoy-config:
```

Comment:
FYI I'm working on the testImage and plugin definitions now. Going to get a few that can run and produce interesting output (job/ds junit/raw passing/failing/etc) and we can expand on it as we want afterwards.